.travis.yml

```diff
@@ -5,11 +5,15 @@ python:
 # command to install dependencies
 install:
   - pip install --upgrade --pre tox
+  - |
+    pip install tox
+    if [[ "${TOXENV#py}" != "$TOXENV" ]]; then
```

Comment:
Pardon my lack of bash-fu, what does this expression inside `if` does?

Reply:
TOXENV starts with "py".

Reply:
Thanks, and what does `"${TOXENV#py}"` do? (I tried googling it but it wasn't entirely obvious what to search for).

Reply:
It substitutes "py" at the beginning.
Yes, shell programming can be ugly.  See "Parameter Expansion" in bash's man page.

Reply:
Thanks!